manager/assets/modext/widgets/core/modx.panel.js

```diff
@@ -415,6 +415,177 @@ Ext.extend(MODx.FormPanel,Ext.FormPanel,{
             }
         });
     }
+
+    /**
+     * @property {Function} onChangeStaticSource - Updates the static file field based
+     * on the chosen source.
+     *
+     * @param {Object} cmp - The media source field's Ext.Component object
+     * @param {String} elType - The MODX element type (i.e., tv, chunk, or snippet)
+     */
+    ,onChangeStaticSource: function(cmp, elType) {
+
+        const   staticFileField = Ext.getCmp(`modx-${elType}-static-file`),
```

Comment:
Why there are so many spaces between const and name? Also I'm more in favour not grouping variables this way (but it might be just personal preference)

Reply:
This is a preference of mine, as I find it a bit easier to read when all the variable names in a block are left aligned.